src/scripts/components/Play.js

```diff
@@ -563,6 +561,10 @@ class Play extends Component<Props, State> {
       poster = video.thumbnails.get(0)
     }
     import('paratii-mediaplayer').then(CreatePlayer => {
+      if (this.player && this.player.destroy) {
+        this.player.destroy()
+      }
+
```

Comment:
Since our player creating is aync, we must destroy the player after the dynamic import of `paratii-mediplayer` is complete.